db/migrate/20221019154329_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Kindly change the name of the foreign key column name to `author` instead of `user`.

Reply:
@gemmen29  Thanks for the review I was getting an error when I tried changing, but let me try again.